.eslintignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+dist
+coverage
+.github
+.cache
+.cache-jest
+reports
+stryker-tmp
```

Comment:
You can remove this since Stryker is not configured into your project.

Reply:
Ok, I will remove it! ^^